src/test/java/com/wikia/webdriver/testcases/globalnavigationtests/GlobalNavigationHubLinks.java

```diff
@@ -30,7 +30,7 @@ public void TestHubLinksInGlobalNav_001_clickHubsLinks() {
             String link = globalNav.getHubLink(hub);
             hub.click();
 
-            new WebDriverWait(driver, 10).until(ExpectedConditions.urlToBe(link));
+            //new WebDriverWait(driver, 10).until(ExpectedConditions.urlToBe(link));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)